demos/python_demos/multi_camera_multi_person_tracking/multi_camera_multi_person_tracking.py

```diff
@@ -0,0 +1,146 @@
+                self.process = False
+                break
+            if has_frames:
+                self.frames_queue.put(frames)
+
+
+def run(params, capture, detector, reid):
+    win_name = 'Multi camera tracking'
+    config = {}
+    if len(params.config):
+        config = read_py_config(params.config)
+
+    tracker = MultiCameraTracker(capture.get_num_sources(), reid, **config)
+
+    thread_body = frames_thread_body(capture, max_queue_length=len(capture.captures) * 2)
+    frames_thread = Thread(target=thread_body)
+    frames_thread.start()
+
+    if len(params.output_video):
+        video_output_size = (1920 // capture.get_num_sources(), 1080)
+        fourcc = cv.VideoWriter_fourcc(*'XVID')
+        output_video = cv.VideoWriter(params.output_video,
+                                      fourcc, 24.0,
+                                      video_output_size)
+    else:
+        output_video = None
+
+    while cv.waitKey(1) != 27 and thread_body.process:
+        start = time.time()
+        try:
+            frames = thread_body.frames_queue.get_nowait()
+        except queue.Empty:
+            frames = None
+
+        if frames is None:
+            continue
+
+        all_detections = detector.get_detections(frames)
+        all_masks = [[] for _ in range(len(all_detections))]
+        for i, detections in enumerate(all_detections):
+            all_detections[i] = [det[0] for det in detections]
+            all_masks[i] = [det[2] for det in detections if len(det) == 3]
+
+        tracker.process(frames, all_detections, all_masks)
+        tracked_objects = tracker.get_tracked_objects()
+
+        fps = round(1 / (time.time() - start), 1)
+        vis = visualize_multicam_detections(frames, tracked_objects, fps)
+        cv.imshow(win_name, vis)
+        if output_video:
+            output_video.write(cv.resize(vis, video_output_size))
+
+    thread_body.process = False
+    frames_thread.join()
+
+    if len(params.history_file):
+        history = tracker.get_all_tracks_history()
+        with open(params.history_file, 'w') as outfile:
+            json.dump(history, outfile)
+
+
+def main():
+    """Prepares data for the person recognition demo"""
+    parser = argparse.ArgumentParser(description='Multi camera multi person \
+                                                  tracking live demo script')
+    parser.add_argument('--videos', type=str, nargs='+', help='Input videos')
+    parser.add_argument('--cam_ids', type=int, nargs='+', default=[-1],
+                        help='Indexes of input cameras')
+
+    parser.add_argument('--pd_model', type=str, required=True)
+    parser.add_argument('--pd_thresh', type=float, default=0.6,
+                        help='Threshold for person detection model')
+
+    parser.add_argument('--pr_model', type=str, required=False)
+    parser.add_argument('--pr_thresh', type=float, default=0.6,
+                        help='Threshold for person re-identification model')
```

Comment:
Most demos use `-reid` key independently of whether it is for face or person. I think we should follow that style.

Reply:
Fixed

Reply:
I am sorry for placing a misleading comment. Normally the convention is:
use `-m` for the primary model (detection in this case)
`-m_suffix` for the rest models. In this case it should be `-m_reid`

By the way the same could be applied for device if it was possible to configure device for each network separately. In this case it should be `-d`, `--device`.